### PR TITLE
chore: Add example of a form page

### PR DESCRIPTION
**Pull request by VincentSmedinga**
Copied from the Signalen prototype, and:

- added a text area
- translated field names from Dutch to English
- added autocomplete values

### CODE REVIEW

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                      <Label htmlFor="countryCode">Landnummer</Label>
+                      <Select name="countryCode">
+                        <option value="+31">Nederland +31</option>
+                        <option value="+32">België +32</option>
+                        <option value="+33">Frankrijk +33</option>
+                      </Select>
+                    </Field>
+                    <Field>
+                      <Label htmlFor="phone">Telefoonnummer</Label>
+                      <TextInput autoComplete="phone" id="phone" name="phone" />
```
Comment (alimpens) [inline storybook/src/pages/amsterdam/FormPage/FormPage.tsx:92]:
```suggestion
                      <TextInput autoComplete="tel" id="phone" name="phone" />
```

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      <SkipLink href="#main">Direct naar inhoud</SkipLink>
+      <Screen maxWidth="wide">
+        <AppHeader />
+        <Grid paddingTop="small" paddingBottom="medium">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+            <Breadcrumb>
+              <Breadcrumb.Link>Home</Breadcrumb.Link>
+            </Breadcrumb>
+            <form className="ams-gap--md" id="main" onSubmit={(e) => e.preventDefault()}>
+              <Heading>Contact</Heading>
```
Comment (alimpens) [inline storybook/src/pages/amsterdam/FormPage/FormPage.tsx:38]: [NLDS recommends placing information about where the user is outside of the `<form>` tag](https://nldesignsystem.nl/richtlijnen/formulieren/meerdere-stappen#plaats-de-informatie-over-waar-de-gebruiker-is-in-de-stappen-boven-het-formulier)

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                  </Row>
+                </Column>
+              </FieldSet>
+              <Alert severity="info">
+                <Paragraph>
+                  We bewaren uw contactgegevens voor het afhandelen van uw vraag of klacht en het verbeteren van onze
+                  dienstverlening.
+                </Paragraph>
+              </Alert>
+              <Row>
```
Comment (alimpens) [inline storybook/src/pages/amsterdam/FormPage/FormPage.tsx:103]: Just a `div` is enough here, no?

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                    </Radio>
+                    <Radio name="city" value="anders">
+                      Anders
+                    </Radio>
+                  </Field>
+                  <Field>
+                    <Label htmlFor="email">E-mail</Label>
+                    <TextInput autoComplete="email" id="email" name="email" />
+                  </Field>
+                  <Row>
```
Comment (alimpens) [inline storybook/src/pages/amsterdam/FormPage/FormPage.tsx:81]: The alignment between the 2 fields is off on smaller screens

Comment (VincentSmedinga) [reply storybook/src/pages/amsterdam/FormPage/FormPage.tsx:80]: I’ve added a `wrap` prop to the `Row` component for this: #1319.

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx (hunk 3 above)
Comment (VincentSmedinga) [reply storybook/src/pages/amsterdam/FormPage/FormPage.tsx]: I like the `Row` better, but yes, a `div` is enough.

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                </Paragraph>
+                <TextArea
+                  aria-describedby="bodyDescription"
+                  id="body"
+                  onChange={(e) => setTextareaLength(e.target.value.length)}
+                  rows={4}
+                />
+                <FormFieldCharacterCounter length={textareaLength} maxLength={1000} />
+              </Field>
+              <FieldSet aria-describedby="contactDetailsDescription" legend="Wat zijn uw gegevens?">
```
Comment (dlnr) [inline storybook/src/pages/amsterdam/FormPage/FormPage.tsx:52]: If your focus is on the first input the describedby property is not read.

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+                  </Row>
+                </Column>
+              </FieldSet>
+              <Alert severity="info">
+                <Paragraph>
+                  We bewaren uw contactgegevens voor het afhandelen van uw vraag of klacht en het verbeteren van onze
+                  dienstverlening.
+                </Paragraph>
+              </Alert>
+              <div>
```
Comment (dlnr) [inline storybook/src/pages/amsterdam/FormPage/FormPage.tsx:102]: Perhaps a `Row` is recommended to anticipate multiple buttons with some spacing.

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+                  </FieldSet>
+                  <Field>
+                    <Label htmlFor="email">E-mail</Label>
+                    <TextInput autoComplete="email" id="email" name="email" />
+                  </Field>
+                  <Row>
+                    <Field>
+                      <Label htmlFor="countryCode">Landnummer</Label>
+                      <Select name="countryCode">
+                        <option value="+31">Nederland +31</option>
```
Comment (dlnr) [inline storybook/src/pages/amsterdam/FormPage/FormPage.tsx:84]: We have `Select.Option` child component. This is probably a good example for reusing stories.

File: storybook/src/pages/amsterdam/FormPage/FormPage.tsx (hunk 5 above)
Comment (VincentSmedinga) [reply storybook/src/pages/amsterdam/FormPage/FormPage.tsx:52]: VoiceOver on my iPhone keeps reading out that description with every label or input I focus on. Seems like we need to test this further. In this PR or a new one? cc @alimpens

Comment (dlnr) [reply storybook/src/pages/amsterdam/FormPage/FormPage.tsx:52]: Perhaps leave it out for now then?

Comment (VincentSmedinga) [reply storybook/src/pages/amsterdam/FormPage/FormPage.tsx:52]: I think we should. It’s very important but makes the scope of this PR much larger. I added a [ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-862) to our next sprint.

**Review by alimpens — APPROVED** [review]

**Pull request merged**